bee-api/bee-rest-api/src/endpoints/config.rs

```diff
@@ -2,83 +2,56 @@
 
+/// Default REST API binding address.
 pub(crate) const DEFAULT_BIND_ADDRESS: &str = "/ip4/0.0.0.0/tcp/14265";
-
-// all available routes
-pub(crate) const ROUTE_ADD_PEER: &str = "/api/v1/peers";
-pub(crate) const ROUTE_BALANCE_BECH32: &str = "/api/v1/addresses/:address";
-pub(crate) const ROUTE_BALANCE_ED25519: &str = "/api/v1/addresses/ed25519/:address";
-pub(crate) const ROUTE_HEALTH: &str = "/health";
-pub(crate) const ROUTE_INFO: &str = "/api/v1/info";
-pub(crate) const ROUTE_MESSAGE: &str = "/api/v1/messages/:messageId";
-pub(crate) const ROUTE_MESSAGE_CHILDREN: &str = "/api/v1/messages/:messageId/children";
-pub(crate) const ROUTE_MESSAGE_METADATA: &str = "/api/v1/messages/:messageId/metadata";
-pub(crate) const ROUTE_MESSAGE_RAW: &str = "/api/v1/messages/:messageId/raw";
-pub(crate) const ROUTE_MESSAGES_FIND: &str = "/api/v1/messages";
-pub(crate) const ROUTE_MILESTONE: &str = "/api/v1/milestones/:milestoneIndex";
-pub(crate) const ROUTE_MILESTONE_UTXO_CHANGES: &str = "/api/v1/milestones/:milestoneIndex/utxo-changes";
-pub(crate) const ROUTE_OUTPUT: &str = "/api/v1/outputs/:outputId";
-pub(crate) const ROUTE_OUTPUTS_BECH32: &str = "/api/v1/addresses/:address/outputs";
-pub(crate) const ROUTE_OUTPUTS_ED25519: &str = "/api/v1/addresses/ed25519/:address/outputs";
-pub(crate) const ROUTE_PEER: &str = "/api/v1/peers/:peerId";
-pub(crate) const ROUTE_PEERS: &str = "/api/v1/peers";
-pub(crate) const ROUTE_REMOVE_PEER: &str = "/api/v1/peers/:peerId";
-pub(crate) const ROUTE_SUBMIT_MESSAGE: &str = "/api/v1/messages";
-pub(crate) const ROUTE_SUBMIT_MESSAGE_RAW: &str = "/api/v1/messages";
-pub(crate) const ROUTE_TIPS: &str = "/api/v1/tips";
-pub(crate) const ROUTE_RECEIPTS: &str = "/api/v1/receipts";
-pub(crate) const ROUTE_RECEIPTS_AT: &str = "/api/v1/receipts/:milestoneIndex";
-pub(crate) const ROUTE_TREASURY: &str = "/api/v1/treasury";
-pub(crate) const ROUTE_TRANSACTION_INCLUDED_MESSAGE: &str = "/api/v1/transactions/:transactionId/included-message";
-pub(crate) const ROUTE_WHITE_FLAG: &str = "/api/plugins/debug/whiteflag";
-
-/// the routes that are available for public use
-pub(crate) const DEFAULT_PUBLIC_ROUTES: [&str; 21] = [
-    ROUTE_BALANCE_BECH32,
-    ROUTE_BALANCE_ED25519,
-    ROUTE_HEALTH,
-    ROUTE_INFO,
-    ROUTE_MESSAGE,
-    ROUTE_MESSAGE_CHILDREN,
-    ROUTE_MESSAGE_METADATA,
-    ROUTE_MESSAGE_RAW,
-    ROUTE_MESSAGES_FIND,
-    ROUTE_MILESTONE,
-    ROUTE_MILESTONE_UTXO_CHANGES,
-    ROUTE_OUTPUT,
-    ROUTE_OUTPUTS_BECH32,
-    ROUTE_OUTPUTS_ED25519,
-    ROUTE_SUBMIT_MESSAGE,
-    ROUTE_SUBMIT_MESSAGE_RAW,
-    ROUTE_TIPS,
-    ROUTE_RECEIPTS,
-    ROUTE_RECEIPTS_AT,
-    ROUTE_TREASURY,
-    ROUTE_TRANSACTION_INCLUDED_MESSAGE,
-];
-pub(crate) const DEFAULT_ALLOWED_IPS: [IpAddr; 2] = [
-    IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1)),
-    IpAddr::V6(Ipv6Addr::new(0, 0, 0, 0, 0, 0, 0, 1)),
+/// Default JWT salt for REST API.
+pub(crate) const DEFAULT_JWT_SALT: &str = "Bee";
```

Comment:
We already have a constant holding "Bee" in `bee-node` I think. Should we use that instead, otherwise we'ld have to make sure those 2 constants remain in sync, or would that not pose a real problem?

Reply:
ah, that will cause a circular dep, I guess. Maybe we should move this constant to `bee-identity` eventually to have only one constant holding "Bee" :thinking: